.changeset/config.json

```diff
@@ -1,6 +1,9 @@
 {
   "$schema": "https://unpkg.com/@changesets/config@1.5.0/schema.json",
-  "changelog": "@changesets/cli/changelog",
+  "changelog": [
+    "@changesets/changelog-github",
+    { "repo": "graphql/graphiql" }
+  ],
```

Comment:
This just formats the changelog entries a little nicer, puts in @<author> tags and PR numbers and other niceties. Most OSS projects that use Changesets use this.

cc @Andarist @mitchellhamilton

Reply:
Yep - this is a cool generator that I'm recommending to be used by default. 👍

Reply:
sounds great! can you open another PR to add this?